CMakePresets.json

```diff
@@ -67,6 +67,17 @@
       "inherits": [
         "base"
       ]
+    },
+    {
+      "name": "iwyu",
+      "displayName": "IWYU",
+      "description": "Fix includes with include what you use",
+      "cacheVariables": {
+        "ENABLE_IWYU": "ON"
+      },
+      "inherits": [
+        "base"
```

Comment:
```suggestion
        "default"
```

Reply:
Hmm, won't this use RelWithDebInfo build type instead of Debug?